pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -32,7 +32,10 @@ type ClusterServiceBroker struct {
 	// Non-namespaced.  The name of this resource in etcd is in ObjectMeta.Name.
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   ClusterServiceBrokerSpec   `json:"spec"`
+	Spec ClusterServiceBrokerSpec `json:"spec"`
+
+	// Status represents the current status of a broker.
+	// +optional
```

Comment:
We should go ahead and put `+optional` on all of the `TypeMeta`, `ObjectMeta`, Spec, and Status fields of each resource type while we are at it. Along with that we should also add the `omitempty` struct tag.

Reply:
Done. PTAL